curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa2943669c9d5026af6985.md

```diff
@@ -7,15 +7,19 @@ dashedName: step-172
 
 # --description--
 
-The `indexOf()` array method returns the first index at which a given element can be found in the array, or `-1` if the element is not present.
+The `.includes()` method determines if an array contains an element and will return either `true` or `false`.
 
-After your `for` loop, add an `if` statement to check if the `guess` is in the `numbers` array. You can check if `indexOf` is not equal (`!==`) to `-1`.
+After your `for` loop, add an `if` statement to check if the `guess` is in the `numbers` array. You can use the `.includes()` method to check if the array contains the `guess`.
 
-Here is an example of the `indexOf` syntax:
+Here is an example of the `.includes()` syntax:
 
 ```js
-const numbers = [1, 2, 3];
-numbers.indexOf(2) // 1
+const numbersArray = [1, 2, 3, 4, 5]
+const number = 3
+
+if (numbersArray.includes(number)) {
+  console.log("The number is in the array.")
+}
 ```
```

Comment:
I think we should move the example up before the requirement.

---

The `.includes()` method determines if an array contains an element and will return either `true` or `false`.

Here is an example of the `.includes()` syntax:

```js
const numbersArray = [1, 2, 3, 4, 5]
const number = 3

if (numbersArray.includes(number)) {
  console.log("The number is in the array.")
}
```

After your `for` loop, add an `if` statement to check if the `guess` is in the `numbers` array. You can use the `.includes()` method to check if the array contains the `guess`.

Reply:
Sounds good